plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -406,4 +407,36 @@
         </li>
     </xsl:template>
 
+
+    <!-- adding all borders to choicetables  -->
+    <xsl:template match="*[contains(@class,' task/choicetable ')]" name="topic.task.choicetable">
+        <xsl:apply-templates select="*[contains(@class,' ditaot-d/ditaval-startprop ')]" mode="out-of-line"/>
+        <xsl:call-template name="setaname"/>
+        <table border="1" frame="all" rules="rows" cellpadding="4" cellspacing="0" summary="" class="choicetableborder">
```

Comment:
Can we use CSS instead of XSLT customization? Will it work if we specify borders using 'choicetableborder' class?